Mohammad-Husam/coding_challenge
# Feature/design phase two

**Background**

-  Building chart box and slider

**Modifications**

- Building char box and make it clickable by adding it into Inkwell to handle clicks.
- Building Slider widget and put it into Slider theme widget to customize thumb icon color and design.

**Step to test**

1. open the app.
2. click on the left or right chart box to see the hover action.
3. press on the thumb icon in the slider widget and swipe it
4. observe the result.
